chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/listing/normalizer/CertificationCriterionNormalizer.java

```diff
@@ -27,52 +23,18 @@
 @Component
 public class CertificationCriterionNormalizer {
 
-    private CertificationCriterionDAO criterionDao;
     private CertificationResultRules certResultRules;
 
     @Autowired
-    public CertificationCriterionNormalizer(CertificationCriterionDAO criterionDao,
-            CertificationResultRules certResultRules) {
-        this.criterionDao = criterionDao;
+    public CertificationCriterionNormalizer(CertificationResultRules certResultRules) {
         this.certResultRules = certResultRules;
     }
 
     public void normalize(CertifiedProductSearchDetails listing) {
-        addEditionCriteriaNotPresentInListing(listing);
+        listing.getCertificationResults().removeIf(cr -> cr.isSuccess() == null || !cr.isSuccess());
```

Comment:
Does this normalizer run on 2014 Edition Listings when/if they're edited? Because if it does, there are some 2014 Criteria that have g1Success or g2Success values as "true" that still need to show on the Listing, even if the Listing doesn't attest to the criteria

Reply:
It only runs on uploaded listings.

Reply:
I'm pretty sure the normalizers all run on both uploaded and editing listings. I see the ListingDetailsNormalizer.normalize method being called on upload and update.